tests/e2e/standard-config/create-place-sms.wdio-spec.js

```diff
@@ -0,0 +1,44 @@
+const smspage = require('../../page-objects/admin/sms.wdio.page');
+const utils = require('../../utils');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const contactPage = require('../../page-objects/contacts/contacts.wdio.page');
+const usersAdminPage = require('../../page-objects/admin/user.wdio.page');
+
+const messageValue = 'N Potu';
+const phoneNumber = '+12068881234';
+
+const placeFactory = require('../../factories/cht/contacts/place');
+const personFactory = require('../../factories/cht/contacts/person');
+const places = placeFactory.generateHierarchy(); // This generates ['district_hospital', 'health_center', 'clinic']
+const sittuPerson = personFactory.build(
+  {
+    name: 'Sittu',
+    phone: phoneNumber,
+    parent: {
+      _id: places[0]._id,
+      parent: places[0].parent
+    }
+  });
+
+describe('SMS Test Forms', async () => {
+  beforeEach(async () => {
+    await utils.saveDocs([...places, sittuPerson]);
+    await loginPage.cookieLogin();
+  });
+
+  it('create person via SMS', async () => {    
```

Comment:
I think this test needs to be reworked a bit and the functionality of the test is wasted on testing some admin page screens that are not the core of what this test case should be. 

You can avoid the UI all together on this and setup via the API by adding it to your `saveDocs` call above. 
```
    await usersAdminPage.goToAdminUser();
    await usersAdminPage.openAddUserDialog();
    await usersAdminPage.inputAddUserFields('sittu', 'Sittu', 'Regional manager - restricted to their place',
      places[0].name, sittuPerson.name, 'Pass@500', phoneNumber);
    await usersAdminPage.saveUser();
```

This is just a test page for whomever wants to do it via UI quickly. We can use the API to create SMS messages as we would here. Gets rid of another interaction with the UI. 
```
    await smspage.goToAdminSms();
    await smspage.inputMessageInfoAndSubmit(messageValue, phoneNumber);
```

Here is the API doc to interact with. 

https://docs.communityhealthtoolkit.org/apps/reference/api/#post-apisms